conda_forge_tick/utils.py

```diff
@@ -543,3 +544,27 @@ def as_iterable(iterable_or_scalar):
         return iterable_or_scalar
     else:
         return (iterable_or_scalar,)
+
+
+def mamba_cant_solve(packages, os_arch):
+
+    subprocess_list = ['mamba', 'install'] + packages + ['-c', f'https://conda.anaconda.org/conda-forge/{os_arch}', '--override-channels']
+    results = subprocess.check_output(subprocess_list)
```

Comment:
I think if you pass in `universal_newlines=True` it comes back as a string rather than bytes